### PR TITLE
Layout and Column Components are tested.

### PR DESCRIPTION
**Pull request by patrickkim**

### CODE REVIEW

**Comment by drewdrewthis** [comment]
@patrickkim Please squash

File: src/atoms/Layout/__tests__/layout.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    expect(wrapper.find('Col').get(0).props.largeCols).to.equal(7);
+    expect(wrapper.find('Col').get(1).props.largeCols).to.equal(5);
+  });
+
+  it('wraps its child `<Col>` with column sizes', () => {
+    const wrapper = shallow(
+      <Layout
+        smallCols={[ 1 ]}
+      >
+        <div className='child' />
```
Comment (drewdrewthis) [inline src/atoms/Layout/__tests__/layout.spec.js]: @patrickkim Do we need this `className` here?

File: src/atoms/Layout/__tests__/layout.spec.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        <Col smallCols={3}>
+          <div className='one-off' />
```
Comment (drewdrewthis) [inline src/atoms/Layout/__tests__/layout.spec.js]: Same here. Do the tests need these decorative classNames?

**Comment by drewdrewthis** [comment]
@patrickkim Sorry -- I didn't realize you had already merged it. I thought you were waiting for a CR?

File: src/atoms/Layout/__tests__/layout.spec.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
+
+describe('Layout', () => {
+
+  it('is a <Layout> tag', () => {
+    const wrapper = shallow(<Layout />);
+
+    expect(wrapper.length).to.equal(1);
```
Comment (patrickkim) [inline src/atoms/Layout/__tests__/layout.spec.js]: Fair point. Was going off the base. Changing the base test template.

File: src/atoms/Layout/__tests__/layout.spec.js (hunk 2 above)
Comment (patrickkim) [inline src/atoms/Layout/__tests__/layout.spec.js]: They don't. As I was debugging I found them useful.

**Comment by patrickkim** [comment]
All Good. It didn't change or add anything new & just suite of tests so I didn't deem it necessary for a CR.

**Pull request merged**